py3status/core.py

```diff
@@ -677,6 +678,21 @@ def setup(self):
             # load and spawn i3status.conf configured modules threads
             self.load_modules(self.py3_modules, user_modules)
 
+        # determine the target output format (i3bar or tmux)
+        self.output_format = self.config["py3_config"]["general"]["output_format"]
```

Comment:
Does `output_format` still exist if unspecified?

Reply:
yes, it defaults to "i3bar" as per `constants.py`